src/web/client/common/constants.ts

```diff
@@ -116,6 +116,9 @@ export const pathParamToSchema = new Map([
 
 export enum queryParameters {
     ORG_URL = 'orgUrl',
+    ORG_ID = 'organizationId',
```

Comment:
pac CLI already has those 2 guids (and we have the proper data tagging and GDPR in place), good to go.